cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -276,6 +276,25 @@
 
       public static final String SERVICE_DESCRIPTION = "Service to perform dataset operations.";
     }
+
+    /**
+     * Dataset extensions.
+     */
+    public static final class Extensions {
+      public static final String EXT_DIR = "dataset.extensions.dir";
```

Comment:
Since constants defined in this class is already under `Extensions`, no need to prefix constant name with `EXT` or `EXTENSIONS`. When being used, it'll be more fluent, e.g. `Constants.Extensions.DIR` instead of `Constants.Extensions.EXT_DIR`.